source/scss/_library/_objects.forms.scss

```diff
@@ -12,3 +12,114 @@
 .is-valid {
   border-color: $c-valid !important;
 }
+
+/**
+ * Search entry
+ */
+.c-search {
+  display: flex;
+  justify-content: space-between;
+
+  &__button {
+    height: 100%;
+    padding: 10px;
+  }
+}
+
+// Heading search menu
+.c-search--top {
+  height: 50px;
+  width: 265px;
+  position: absolute;
+  right: 0;
+  top: 0;
+  z-index: -1;
+  overflow: hidden;
+  transition: z-index .1s ease-out .5s;
+
+  @include media('<=small') {
+    display: none;
+  }
+
+  .c-search__close {
+    position: absolute;
+    left: $space-half;
+    top: 50%;
+    width: 24px;
+    height: 24px;
+    transform: translate(-40px, -50%);
+    border-radius: 50%;
+    background-color: $c-gray--light;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    transition: transform .15s ease-out;
+    z-index: 10;
+    svg {
+      width: 10px;
+      height: 10px;
+    }
+    path {
+      fill: $c-reddish-orange;
+    }
+  }
+
+  .c-search__input {
```

Comment:
I suppose this is more of a laura/chris question, but we'll need to set the font for the input field itself too

Reply:
I have that default set [right here](https://github.com/nypublicradio/pattern-lab/pull/2/files#diff-cf5e92a0528944bfa668fedf287717e6R36), but we can update as needed. This one seemed to be different from the rest.